unit_tests.py

```diff
@@ -215,7 +215,7 @@ def test_build_multi_fasta(self, _, mock_extract_s3_bucket, mock_extract_s3_key)
         mock_open().read.side_effect=["AAA\nAAA", "TTT\nTTT", "CCC\nCCC", "GGG\nGGG"]
         # run build_multi_fasta() with test_df and a patched open
         with mock.patch("builtins.open", mock_open):
-            btb_phylo.build_multi_fasta("foo", test_df)
+            btb_phylo.build_multi_fasta("foo", test_df, '/mnt/fsx-017/phyloConsensus/')
```

Comment:
Can you just test that this works with a general filepath like `"foo"`, instead of the `fsx-017`? The unittest should work in circleCI where this fsx isn't mounted

Reply:
Although, actually we know the work because the circleci passed. But could you still just change to something more generic.